ui/app/css/itcss/components/sections.scss

```diff
@@ -309,7 +309,8 @@ textarea.twelve-word-phrase {
 }
 
 .buy-inputs {
-  font-size: 13px;
+  @include H6;
```

Comment:
This appears to be an old, unused style.

Reply:
So for these unused styles, were you planning on updating the PR to remove them altogether? Or would you prefer to remove them in a separate PR? Just wondering so that I can help ensure we don't forget to remove them.

Reply:
These were removed in #9146